docs/src/usage/analysis.md

```diff
@@ -165,12 +165,46 @@ tsc_fig = ADRIA.viz.ts_cluster(
     axis_opts=axis_opts
 )
 
-# save final figure
+# Save final figure
 save("tsc.png", tsc_fig)
 ```
 
 ![Plots of Temporal Sensitivities](/ADRIA.jl/dev/assets/imgs/tsc.png?raw=true "Time Series Cluster")
 
+## Time Series Clustering Map
+
+When using Time Series Clustering to cluster among multiple sites using some metric, it is 
+possible to visualize the result as a map.
+
+```julia
+# Load data for a spatial domain
+# dom = ADRIA.load_domain("path to domain data")
+
+# Generate scenarios
+# scens = ADRIA.sample(dom, 128)
+
+# Run sampled scenarios for multiple RCP (Representative Concentration Pathway)
+rcp = "45"
+rs = ADRIA.run_scenarios(scens, dom, rcp)
+
+# Extract metric from scenarios
+tac = ADRIA.metrics.total_absolute_cover(rs)
+
+# Summarize scenarios for each site
+m_tac = dropdims(median(tac, dims=:scenarios), dims=:scenarios)
```

Comment:
There's now a `per_site` method

https://github.com/open-AIMS/ADRIA.jl/blob/a550e03c6032938c8f9f9f8a57871100413782af/src/metrics/site_level.jl#L4-L17

(part of the other PR that this is rebased on)

I'll have to finalize the documentation for that though.

Reply:
@ConnectedSystems I tried to use this method but apparently it returns a VECTOR, not a MATRIX. If I understood correctly, this is computing the metric over scenarios AND timesteps? If this is true, we may need to change this method..?

Reply:
Yes, it returns a vector providing a value for each site (as in the documentation), which is what you want when showing outcomes spatially (e.g., the mean outcomes over time, or the outcome at a point in time).

If you want the "raw" data for each site AND timestep, you're looking for the relevant metric functions:

https://github.com/open-AIMS/ADRIA.jl/blob/5e97710e8fd5f40ab35c26cc482be47fde7e8694/src/metrics/metrics.jl#L161-L173

Reply:
But the thing is, what this line of code is doing:

`m_tac = dropdims(median(tac, dims=:scenarios), dims=:scenarios)`

I can't do with `per_site` - right? 
When you wrote this comment, how were you thinking I could use this method `per_site` here?

Reply:
You can use `per_site` here? I'm not sure I understand what the issue is.

`m_tac` here is a vector, of $N$ elements, where $N$ is the number of locations, no?

Reply:
`m_tac = ADRIA.metrics.per_site(median, tac)`

Reply:
Oh, I see what you're getting at

Sorry, I was thinking of this line:

```
# Plot figure
tsc_map_fig = ADRIA.viz.map(rs, m_tac, loc_clusters)
```

But internally this already does the `dropdims()`

https://github.com/open-AIMS/ADRIA.jl/blob/ed18747893f03769d24f5da7c81741068b9b2dc6/ext/AvizExt/viz/clustering.jl#L97

So we'd want to replace that with a call to `per_site()`

Sorry for the confusion

Reply:
When I do `m_tac = dropdims(median(tac, dims=:scenarios), dims=:scenarios)` what I get is a: 
```
2-dimensional NamedDimsArray(KeyedArray(...)) with keys:
↓   timesteps ∈ 75-element Vector{Any}
→   sites ∈ 216-element Vector{Any}
```

Reply:
Sorry again for the confusion.

I've added a `loc_trajectory()` method using the same approach. This gets you what you need, and it's much faster than the previous approach.

Please also note I've updated `per_site` to `per_loc` (`loc` being short for "location").
I'm slowly standardizing away from "site" as this terminology is confusion.